types/index.test-d.ts

```diff
@@ -1,43 +1,43 @@
-import { expectError, expectType } from 'tsd'
-import { clamp, Email, isEmail, round } from '.'
```

Comment:
Rather odd to see this `'.'`  import in a package that declares `"type": "module"`. (The extension-less imports should not be allowed, I think.)